cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -313,17 +312,7 @@ private boolean isIdeHome(Path dir) {
 
   private Path getParentPath(Path dir) {
 
-    try {
-      Path linkDir = dir.toRealPath();
-      if (!dir.equals(linkDir)) {
-        return linkDir;
-      } else {
-        return dir.getParent();
-      }
-    } catch (IOException e) {
-      throw new IllegalStateException(e);
-    }
-
+    return dir.getParent();
```

Comment:
Correct fix. However, having a private method that just wraps an existing method does not add any value.
IMHO we should now remove `getParentPath` method and call `getParent()` directly.